plugin/evm/message/message.go

```diff
@@ -22,8 +22,14 @@ const (
 )
 
 var (
-	_ GossipMessage = AtomicTxGossip{}
-	_ GossipMessage = EthTxsGossip{}
+	_ GossipMessage = AtomicTxGossip{} // TODO deprecate
+	_ GossipMessage = EthTxsGossip{}   // TODO deprecate
```

Comment:
nit: consider using the (*type)(nil) format for all of these type assertions

Reply:
Leaving these as-is for now since it's existing code, we'll just remove these in a future PR when these gossip message types are deprecated.